extensions/azure/blobstorage/blob-core/src/main/java/org/eclipse/dataspaceconnector/azure/blob/core/BlobStoreCoreExtension.java

```diff
@@ -35,9 +35,10 @@ public String name() {
 
     @Override
     public void initialize(ServiceExtensionContext context) {
-        var blobstoreEndpoint = context.getSetting(EDC_BLOBSTORE_ENDPOINT, null);
+        var blobstoreEndpointTemplate = context
+                .getSetting(EDC_BLOBSTORE_ENDPOINT_TEMPLATE, "https://%s.blob.core.windows.net");
```

Comment:
What is the value of using this "template" instead of the endpoint directly? Why did you need to do that?

Reply:
When the provider in system tests copies from source storage account to destination storage account in emulator, the endpoints are different (`"http://127.0.0.1:10000/" + accountName`)